srcpkgs/bitcoin/template

```diff
@@ -1,39 +1,49 @@
 # Template file for 'bitcoin'
 pkgname=bitcoin
-version=0.19.1
-revision=4
+version=0.21.0
+revision=1
 build_style=gnu-configure
 configure_args="--with-incompatible-bdb --disable-ccache --disable-static
- --enable-hardening --with-boost=${XBPS_CROSS_BASE}/usr"
-hostmakedepends="pkg-config yasm"
-makedepends="db-devel boost-devel miniupnpc-devel
- libevent-devel qrencode-devel libatomic-devel"
+ --enable-hardening --with-boost=${XBPS_CROSS_BASE}/usr --with-gui"
+hostmakedepends="automake libtool pkg-config yasm"
+makedepends="db-devel boost-devel miniupnpc-devel qt5-tools-devel
+ libevent-devel qrencode-devel libatomic-devel sqlite-devel"
 short_desc="Bitcoin is a peer-to-peer network based digital currency"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="MIT"
 homepage="https://bitcoin.org/"
 distfiles="https://bitcoin.org/bin/bitcoin-core-${version}/bitcoin-${version}.tar.gz
- https://raw.github.com/bitcoin-core/packaging/master/debian/bitcoin-qt.desktop
- https://raw.github.com/bitcoin/bitcoin/v${version}/share/pixmaps/bitcoin128.png"
-checksum="f2591d555b8e8c2e1bd780e40d53a91e165d8b3c7e0391ae2d24a0c0f23a7cc0
+ https://raw.githubusercontent.com/bitcoin-core/packaging/master/debian/bitcoin-qt.desktop
```

Comment:
Please don't get it from mater (it's an error from the previous version).

Reply:
> mater

Do you mean 'master'? Anyway, updated the PR.